WhirlyGlobeSrc/WhirlyGlobeLib/include/DynamicTextureAtlas.h

```diff
@@ -206,7 +206,7 @@ class DynamicTextureAtlas
 
     /// Clear out the active dynamic textures.  Caller deals with the
     ///  change requests.
-    void shutdown(ChangeSet &changes);
+    void teardown(ChangeSet &changes);
```

Comment:
@mousebird it seems to me a bit confusing. DynamicTextureAtlas already has a "cleanup" method, so now we have both "cleanup" and "teardown".
I'd say both methods are quite similar, the only different is "cleanup" only removes the texture if it has regions. Can we merge both methods together in one single "cleanup" method?